uportal-war/src/main/java/org/apereo/portal/portlets/backgroundpreference/BackgroundSetSelectionStrategy.java

```diff
@@ -33,6 +33,9 @@
 
     String[] getImageThumbnailSet(PortletRequest req);
 
+    @since 4.3.1
```

Comment:
@CR96 This PR is targeting `master` (i.e. release `5.0.0`), if it is a candidate to be backported to `rel-4-3-patches` could you make a note of that in the PR's "Description of change" section? Thanks!

Also note that [the `4.3.1` release](https://github.com/Jasig/uPortal/releases/tag/uportal-4.3.1) has been cut, this change would become a part of release `4.3.2` or release `4.4.0`.